docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -684,7 +684,11 @@ For example, you can configure a set of FHIRPath Constraints to run for resource
 }
 ```
 
-See the [Validation Guide - Built-in profile support](https://ibm.github.io/FHIR/guides/FHIRValidationGuide#built-in-profile-support) for a list of pre-packaged Implmentation Guide resources and see [Validation Guide - Making profiles available to the fhir registry](https://ibm.github.io/FHIR/guides/FHIRValidationGuide#making-profiles-available-to-the-fhir-registry-component-fhirregistry) for information about how to extend the server with additional Implementation Guide artifacts.
+See [Validation Guide - Built-in profile support](https://ibm.github.io/FHIR/guides/FHIRValidationGuide#built-in-profile-support) for a list of pre-packaged Implmentation Guide resources.
```

Comment:
minor, but only base spec profiles are built-in (and these are not from any "Implementation Guide").
```suggestion
The IBM FHIR Server pre-packages all conformance resources from the core specification.
```

Reply:
Done.